### ISSUE TITLE
A11y_.NETCore_ASP.NETCore_About_UseOfColor: Only color is used for "About" link and underline is not provided.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspnetcore#49676]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Github Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #DesktopWeb; #BM_.NETCore_Web_Jul23; #.NET Core; #WCAG1.4.1; #Win11; 

### Environment Details:
App name: .NET Core
URL: http://localhost:5268

Chromium Edge: Version 116.0.1938.29 (Official build) dev (64-bit)

### Repro Steps:

1. Open command prompt and enter command "mkdir foldername" e.g. "mkdir BlazorWasm".
2. Move to the created directory using command "cd foldername".
3. Run command "dotnet new blazorwasm" and hit ENTER key.
4. Run command "dotnet run".
5. Copy the First URL link from Command prompt and paste it in URL section of Browser and hit ENTER key.
6. TAB to "About" link and observe that only color is used for "About" link and underline is not provided.

### Actual Result: 
Only color is used for "About" link and underline is not provided.

### Expected Result:
Only color should not be used for "About" link and underline should be provided.

### User Impact:
Color blind users will not be able to differentiate between the normal text and the link.

**Comment by mkArtakMSFT** [comment]
@PoojaNamde  why the `About` link specifically has to have an underline ? This one doesn't make sense frankly.

**Comment by merriemcgaw** [comment]
This is a sev3 usability issue. Someone might not understand that the word "About" is a link. That said, there is no SLA for fixing or addressing Sev3/4 bugs, so it is up to the feature team to decide whether they would like to make a change.

**Comment by PoojaNamde** [comment]
#Regressed:08-17-23; #Closed;

Adding tagging as Po closed this bug.
